src/main/support.js

```diff
@@ -25,43 +23,33 @@ function getSupportInfo(version, opts) {
   const plugins = opts.plugins;
 
   const options = arrayify(
-    Object.assign(
-      plugins.reduce(
-        (currentOptions, plugin) =>
-          Object.assign(currentOptions, plugin.options),
-        {}
-      ),
-      coreOptions
-    ),
+    Object.assign({}, ...plugins.map(({ options }) => options), coreOptions),
     "name"
   )
-    .sort((a, b) => (a.name === b.name ? 0 : a.name < b.name ? -1 : 1))
     .filter(option => filterSince(option) && filterDeprecated(option))
-    .map(mapDeprecated)
-    .map(mapInternal)
```

Comment:
This looked good. Why change?

Reply:
Old one `map` 3 times, generate two extra new array, should merge into one `map`, for better performance.

Reply:
This place is not performance-critical, so please revert.